README.md

```diff
@@ -8,114 +8,187 @@
 
-The `Prefix` property specifies an optional prefix, which is prepended to each metric key, separated by a dot (`<prefix>.<namespace>.<name>`).
+### Default Dimensions
 
-#### Default Dimensions
+The `DefaultDimensions` property can be used to optionally specify a `List<KeyValuePair<string, string>>`, which will be added as dimensions to all data points.
+Dimension keys will be normalized, de-duplicated, and only one dimension value per key will be sent to the server.
+Dimensions set on instruments will overwrite default dimensions if they share the same name after normalization.
+[OneAgent metadata](#export-oneagent-metadata) will overwrite all dimensions described above, but it only uses Dynatrace-reserved keys starting with `dt.*`.
 
-The `DefaultDimensions` property can be used to optionally specify a list of key/value pairs, which will be added as additional dimensions to all data points.
+The reserved dimension `dt.metrics.source=opentelemetry` will automatically be added to every exported metric when using the exporter.
 
-#### OneAgent metadata enrichment
+### Export OneAgent Metadata
 
 If the `OneAgentMetadataEnrichment` property is set to true, the exporter will retrieve host and process metadata from the OneAgent, if available, and set it as dimensions to all exported metrics.
-More information on the underlying feature used by the exporter can be found in
-the [Dynatrace documentation](https://www.dynatrace.com/support/help/how-to-use-dynatrace/metrics/metric-ingestion/ingestion-methods/enrich-metrics/).
+The `EnrichWithOneAgentMetaData` property on the options object can be used to disable OneAgent metadata export.
+If running on a host with a OneAgent, setting this option will instruct the exporter to read and export metadata collected by the OneAgent to the Dynatrace endpoint.
+This option is set to `true` by default.
+If the OneAgent is running locally, but this option is set to false, no OneAgent metadata will be exported.
+More information on the underlying OneAgent feature that is used by the exporter can be found in the
+[Dynatrace documentation](https://www.dynatrace.com/support/help/how-to-use-dynatrace/metrics/metric-ingestion/ingestion-methods/enrich-metrics/).
 
-### Known issues and limitations
+## Known issues and limitations
 
 The OpenTelemetry Metrics SDK currently does not allow exporters to distinguish between values received from counters and those received from observers.
 Counter values are passed to the exporter as deltas to the last export whereas for observers, the current value is reported.
-For this exporter, we decided to properly support counters and thus send the received values marked as deltas, which will lead to wrong values being reported for observers, however.
+For this exporter, we decided to properly support counters and thus send the received values marked as deltas, which will lead to wrong values being reported for observers.
```

Comment:
yes, due to the fact that metrics in .NET has not evolved in a while since it was excluded in v1.0.0. I think it has been redesigned from the ground up since 1.0.0, and will probably be different there, but i didn't yet look into that.